### PR TITLE
Update staking guide

### PR DESCRIPTION
**Pull request by maltekliemann**
Based on #54 (author unresponsive). `prettier` now searches one depth deeper for files, which led to a whole bunch of reformatting.

### CODE REVIEW

File: docs/build/collator/collator-related-operation-guide.md [hunk 1]
```diff
@@ -6,45 +6,45 @@
...
+   1. Choose `Developer`-`Extrinsics`
 
-   3. Select the account you want to be associated
+   1. Select the account you want to be associated
 
-   4. Choose `authorMapping` and `addAssociation(authorId)`
+   1. Choose `authorMapping` and `addAssociation(authorId)`
 
-   5. Fill your author Id and submit transaction
+   1. Fill your author Id and submit transaction
```
Comment (saboonikhil) [inline docs/build/collator/collator-related-operation-guide.md:32]: I am finding something strange here, not sure if its on my end.

All the steps appear to be numbered as 1 here, but on checking the file they seem to be numbered in ascending order. The preview looks as required.

**Comment by saboonikhil** [comment]
@maltekliemann Super work man, I was thinking about that stale PR just last weekend.

File: docs/build/collator/collator-related-operation-guide.md (hunk 1 above)
Comment (maltekliemann) [reply docs/build/collator/collator-related-operation-guide.md:32]: Using `1.` repeatedly is how you get the markdown renderer to automatically enumerate.

File: docs/build/collator/how-to-stake-in-zeitgeist.md [hunk 2]
```diff
@@ -1,46 +1,43 @@
...
 
-6. Copy one of the canditate address you would to stake your tokens
+1. Copy the candidate address that you want to delegate your tokens to
 
 ![](/img/get-candidates-list.png)
 
 ### Get the Candidate Delegation Count
 
-Choose `developer`---`Chain State`
+1. Choose **Developer** > **Chain state**
```
Comment (saboonikhil) [inline docs/build/collator/how-to-stake-in-zeitgeist.md:22]:
```suggestion
1. Choose **Developer** > **Chain state** > **parachainStaking**
```

Comment (saboonikhil) [reply docs/build/collator/how-to-stake-in-zeitgeist.md:22]: Its marked in the diagram though, we have to choose _parachainStaking_ before selecting _candidateInfo(AccountId32)_

Comment (maltekliemann) [reply docs/build/collator/how-to-stake-in-zeitgeist.md:22]: Thanks, I missed that!

**Review by saboonikhil — APPROVED** [review]
Thank you for bringing this up!

**Pull request merged**